parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -259,6 +274,48 @@ public static Statistics toParquetStatistics(org.apache.parquet.column.statistic
     return stats;
   }
 
+  private static boolean shouldIgnoreStatistics(String createdBy) {
+    if (createdBy == null || !createdBy.startsWith("parquet-mr")) {
```

Comment:
Unless I am missing something, this is what current code will do, i.e., ignore the stats when a version of "parquet-mr" writer can't be determined.